src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -561,6 +540,15 @@ curl -H "Content-Type: application/json" -XPUT "http://localhost:9200/nuxeo/_set
 Nuxeo supports two protocols to access the Elasticsearch cluster: the transport client protocol and the Rest client.
 
 ### The REST Client (default)
+
+{{#> callout type='warning' }}
+The High Level REST Client is deprecated in 7.15.0 in favour of the [Java API Client](https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/8.7/index.html).
```

Comment:
We are targeting 7.x server or 8.x with 7.x compatibility mode and we are using the rest client API from open search, what is the point of documenting that elastic has deprecated the lib on 8.x, 
What can a user do about this information ?